changelogs/fragments/122-antsibull-core-sh.yml

```
@@ -0,0 +1,4 @@
+minor_changes:
+  - "Now depends antsibull-core 2.0.0 or newer; antsibull-core 1.x.y is no longer supported (https://github.com/ansible-community/antsibull/pull/514)."
+  - "Antsibull now no longer depends directly on ``sh`` (https://github.com/ansible-community/antsibull/pull/514)."
+  - "Antsibull now uses ``sys.executable`` instead of ``'python'`` to start the build process of the ``ansible`` package (https://github.com/ansible-community/antsibull/pull/514)."
```

I think this makes more sense, but up to you.

```suggestion
  - "Antsibull now uses ``sys.executable`` instead of the first ``'python'`` in ``$PATH`` to call the pypa build tool (https://github.com/ansible-community/antsibull/pull/514)."
```